notesserver/views.py

```diff
@@ -1,13 +1,31 @@
 import traceback
 import datetime
+
+from django.db import connection
+from django.conf import settings
 from rest_framework import status
 from rest_framework.permissions import AllowAny
 from rest_framework.response import Response
-from rest_framework.views import APIView
-from rest_framework.decorators import api_view, permission_classes, authentication_classes
+from rest_framework.decorators import api_view, permission_classes
 
 from elasticsearch.exceptions import TransportError
-from annotator import es
+
+if not settings.ES_DISABLED:
+    from haystack import connections
+
+    def get_es():
+        return connections['default'].get_backend().conn
+else:
+    from mock import Mock
+
+    def get_es():
+        return Mock(
+            ping=lambda: False,
```

Comment:
should this be True instead?  if ES_DISABLED is set deliberately, the heartbeat will always return failures which probably isn't desired since it's not actually a problem.  alternatively, maybe it's better if the es check is skipped entirely in the heartbeat, since it's N/A under that configuration.